src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  userId: number;
+  todo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ userId, todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(userId).then(setUser);
+  }, [userId]);
```

Comment:
The same, add a loading state. Set it when you start loading something, then when you get any response from the API remove it